packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -23,12 +24,11 @@ const getWebSocket = async (url: string) => {
     }
 };
 
-// Recommended values are between 30 and 60 seconds
 // Note: When the browser tab is suspended, the browser will give this suspended tab approximately every 60 seconds time
 // to execute code. This means that the ping message will be sent every 60 seconds in suspended mode. This depends
 // on the browser and can differ.
 const PING_INTERVAL_MS = 45000;
-
+const DEBOUNCE_CLOSE_INTERVAL_MS = 60000;
```

Comment:
Think > 60 might be better, we might risk disconnects from FE. Maybe 70?

Reply:
With the current solution the disconnect from being in suspense is solved.